test/src/sdk-harness/test_projects/storage_vec/u8/mod.rs

```diff
@@ -0,0 +1,163 @@
+mod utils;
+
+use utils::{
+    setup::get_contract_instance,
+    wrappers::{clear, get, insert, is_empty, len, pop, push, remove, swap_remove},
+};
+
+mod success {
```

Comment:
There is a test for: push, pop, remove, swap_remove, insert, len and is_empty but no test for a successful get() or clear(). Those are used in other tests but tests should be atomic to prevent coupling. This means that those two cases should be tested explicitly even if other functions use them.